packages/@lwc/integration-karma/test/component/spellcheck-attribute/index.spec.js

```diff
@@ -71,3 +74,41 @@ describe('setting static values on non custom elements', () => {
         expect(getSpellcheckPropertyValue(elm, 'div.truthy-v')).toBe(true);
     });
 });
+
+describe('dynamically updating the spellcheck attribute', () => {
+    it('sets the initial value', async () => {
+        const elm = createElement('x-dynamic', { is: Dynamic });
+        document.body.appendChild(elm);
+
+        await Promise.resolve();
+        expect(getSpellcheckValue(elm, '.dynamic')).toBe(null);
+        expect(getSpellcheckPropertyValue(elm, '.dynamic')).toBe(true);
+    });
+
+    const valuesAndExpectations = [
+        ['', '', true],
+        [null, null, true],
+        [undefined, null, true],
+        [false, 'false', false],
+        ['false', 'false', false],
+        ['fAlSe', 'fAlSe', false],
+        [true, 'true', true],
+        ['true', 'true', true],
+        [0, '0', true],
+        ['0', '0', true],
+        ['truthy', 'truthy', true],
+    ];
+
+    for (const [setValue, expectedAttrValue, expectedPropValue] of valuesAndExpectations) {
+        it(`sets the value to ${JSON.stringify(setValue)}`, async () => {
+            const elm = createElement('x-dynamic', { is: Dynamic });
+            document.body.appendChild(elm);
+
+            elm.theSpellcheck = setValue;
+
+            await Promise.resolve();
+            expect(getSpellcheckValue(elm, '.dynamic')).toBe(expectedAttrValue);
+            expect(getSpellcheckPropertyValue(elm, '.dynamic')).toBe(expectedPropValue);
+        });
+    }
+});
```

Comment:
Was worried about regressing dynamically updating the `spellcheck` value, so added a bunch of tests.

These tests are just asserting that our static behavior matches our non-static (historical) behavior.